src/gurt/dlog.c

```diff
@@ -405,7 +434,7 @@ d_log_write(char *msg, int len, bool flush)
 	if (mst.log_buf_nob == 0)
 		return 0; /* nothing to write */
 
-	if (mst.log_size + mst.log_buf_nob >= mst.log_size_max) {
+	if (log_exceed_threshold()) {
```

Comment:
minor nit, s/exceed/exceeds.   As it's written, it sounds like "make log exceed threshold" whereas using exceeds sounds like the check it's doing.